parking_space_type: 1  # 0 is vertical, 1 is parallel
# vertical parking
# parking_space:
#   # points: [[23.0, 4.7, 0.0],
#   #          [25.5, 4.7, 0.0],
#   #          [25.5, 0.0, 0.0],
#   #          [23.0, 0.0, 0.0]]
#   points: [[2.3, 4.7, 0.0],
#            [4.8, 4.7, 0.0],
#            [4.8, 0.0, 0.0],
#            [2.3, 0.0, 0.0]]
#   len: 4.7
#   width: 2.5
#   yaw: 1.57

# parallel parking
parking_space:
  # points: [[55.2, 31.5, 0.0],
  #          [57.7, 31.5, 0.0],
  #          [55.2, 25.5, 0.0],
  #          [57.7, 25.5, 0.0]]
  points: [[0, 2.5, 0.0],
           [6, 2.5, 0.0],
           [6, 0.0, 0.0],
           [0.0, 0.0, 0.0]]
  len: 6
  width: 2.5
  yaw: 0.0

car_parameters:
  len: 2.695
  width: 1.449
  axis_len: 1.284
  wheelbase: 1.868
  # wheelbase: 2.132
  max_turn_angle: 0.785
safe_dist: -1